consul_service_mesh_2/step6.md

```diff
@@ -1,11 +1,11 @@
-#### Configure port forwarding for your frontend application
+#### Configure port forwarding for your frontend service
 
-To access the app running inside your COnsul service mesh you will setup a port forwarding.
+To access the app running inside your Consul service mesh, you will setup port forwarding.
 
 `export IP_ADDR=$(hostname -I | awk '{print $1}')`{{execute}}
 
 `kubectl port-forward service/web 9090:9090 --address ${IP_ADDR}`{{execute}}
 
-This will forward port `9090` of `service/web` on port `9090` of your test machine.
+This will forward port `80` of `service/hashicorp-consul-ui` on port `80` of your test machine.
 
```

Comment:
```suggestion
This will forward port `80` of `service/hashicorp-consul-ui` on port `80` of your test machine.

You can now open the [Dashboard](https://[[HOST_SUBDOMAIN]]-9090-[[KATACODA_HOST]].environments.katacoda.com/ui) tab to be redirected to the web application UI.
```